doc/nrf/introduction.rst

```diff
@@ -8,20 +8,20 @@ About the |NCS|
    :depth: 2
 
 The |NCS| enables you to develop applications for nRF52, nRF53, and nRF91 Series devices.
-It is a set of open source projects maintained by Nordic Semiconductor, consisting of several Git repositories publicly hosted on `GitHub`_:
+The SDK consists of a combination of Nordic-developed software and open source projects, hosted as `Git_` repositories in the `nrfconnect GitHub organization`_. Some of the more prominent repositories include:
 
 * `sdk-nrf`_ repository - contains applications, samples, libraries, and drivers that are specifically targeted at Nordic Semiconductor devices.
 * `sdk-nrfxlib`_ repository - contains closed-source libraries and modules in binary format.
   See the :doc:`nrfxlib documentation <nrfxlib:README>`.
+* `sdk-zephyr`_ repository - contains a fork of the `Zephyr`_ project, which provides samples, libraries, and drivers for a wide variety of devices, including Nordic Semiconductor devices.
+  See the :doc:`documentation <zephyr:index>` in Nordic Semiconductor’s Zephyr fork.
 * `sdk-mcuboot`_ repository - contains a fork of the `MCUboot`_ project, which provides a secure bootloader application.
   You can find the fork in :file:`bootloader/mcuboot` after obtaining the |NCS| source code.
   See the :doc:`documentation <mcuboot:index-ncs>` in Nordic Semiconductor’s MCUboot fork.
-* `sdk-zephyr`_ repository - contains a fork of the `Zephyr`_ project, which provides samples, libraries, and drivers for a wide variety of devices, including Nordic Semiconductor devices.
-  See the :doc:`documentation <zephyr:index>` in Nordic Semiconductor’s Zephyr fork.
 
-Every nRF Connect SDK release consists of a combination of these repositories at different revisions.
-The revision of each of those repositories is determined by the current revision of the main (or manifest) repository, ``sdk-nrf``.
-It contains the SDK manifest file that enables you to manage the repositories as one code base with the west tool.
+Every |NCS| release consists of a combination of all repositories at different revisions.
```

Comment:
I think this paragraph might be better placed above the list and becomes the second paragraph in this section.